terraform/modules/data_share_processor/data_share_processor.tf

```diff
@@ -113,8 +113,8 @@ locals {
     ) : (
     var.ingestor_aws_role_arn
   )
-  ingestion_bucket_name       = "${local.resource_prefix}-ingestion"
-  peer_validation_bucket_name = "${local.resource_prefix}-peer-validation"
+  ingestion_bucket_name       = "${local.resource_prefix}-ingestion-a"
```

Comment:
Why is this change here?

Reply:
We had to move the S3 buckets to `us-east-1` to work around a bug in Google's S3 client where they couldn't send content to two different regions (I think they since fixed it but I'm not going to move the buckets back). Then we couldn't create buckets with the same names in `us-east-1` as they already existed in `us-west-1`, so I added this suffix to work around that. As I said in the PR message, we shouldn't take this change in main since it only supports Narnia.